asian-pacific-journal-of-tropical-disease.csl

```diff
@@ -0,0 +1,156 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" default-locale="en-GB">
+  <info>
+    <title>Asian Pacific Journal of Tropical Disease</title>
+    <title-short>APJTD</title-short>
+    <id>http://www.zotero.org/styles/asian-pacific-journal-of-tropical-disease</id>
+    <link href="http://www.zotero.org/styles/asian-pacific-journal-of-tropical-disease" rel="self"/>
+    <link href="http://www.zotero.org/styles/european-journal-of-human-genetics" rel="template"/>
+    <link href="http://www.elsevier.com/journals/asian-pacific-journal-of-tropical-medicine/1995-7645/guide-for-authors" rel="documentation"/>
```

Comment:
ah man.... thanks for spotting that.
There is no guide for the APJTD actually then. 
As an elsevier journal the APJTM should then have one of their house styles, no?